tests/js/spec/views/dashboardsV2/gridLayout/detail.spec.jsx

```diff
@@ -284,58 +285,61 @@ describe('Dashboards > Detail', function () {
       MockApiClient.clearMockResponses();
     });
 
-    it('can remove widgets', async function () {
-      const updateMock = MockApiClient.addMockResponse({
-        url: '/organizations/org-slug/dashboards/1/',
-        method: 'PUT',
-      });
-      wrapper = mountWithTheme(
-        <ViewEditDashboard
-          organization={initialData.organization}
-          params={{orgId: 'org-slug', dashboardId: '1'}}
-          router={initialData.router}
-          location={initialData.router.location}
-        />,
-        initialData.routerContext
-      );
-      await tick();
-      wrapper.update();
-
-      expect(mockVisit).toHaveBeenCalledTimes(1);
-
-      // Enter edit mode.
-      wrapper.find('Controls Button[data-test-id="dashboard-edit"]').simulate('click');
-
-      // Remove the second and third widgets
-      wrapper
-        .find('WidgetCard')
-        .at(1)
-        .find('IconClick[data-test-id="widget-delete"]')
-        .simulate('click');
-
-      wrapper
-        .find('WidgetCard')
-        .at(1)
-        .find('IconClick[data-test-id="widget-delete"]')
-        .simulate('click');
-
-      // Save changes
-      wrapper.find('Controls Button[data-test-id="dashboard-commit"]').simulate('click');
-
-      expect(updateMock).toHaveBeenCalled();
-      expect(updateMock).toHaveBeenCalledWith(
-        '/organizations/org-slug/dashboards/1/',
-        expect.objectContaining({
-          data: expect.objectContaining({
-            title: 'Custom Errors',
-            widgets: [widgets[0]],
-          }),
-        })
-      );
-
-      // Visit should not be called again on dashboard update
-      expect(mockVisit).toHaveBeenCalledTimes(1);
-    });
+    // TODO(nar): Fix in deletion PR
+    // it('can remove widgets', async function () {
```

Comment:
I would delete this commented out block of code since you'll be able to find it in git history anyways 😄 . Makes the code a bit cleaner.